praktikumsplaner-backend/src/test/java/de/muenchen/oss/praktikumsplaner/service/PraktikumsstellenServiceTest.java

```diff
@@ -474,4 +476,37 @@ public void testGetAllAssignedPraktikumsstellenInMostRecentPassedMeldezeitraum()
         assertEquals(5, result.size());
         assertEquals(dtos, result);
     }
+
+    @Test
+    public void testDeletePraktikumsstelle() {
```

Comment:
The methods that you have added are:
- deleteStudiumsPraktikumsstelle
- deleteAusbildungsPraktikumsstelle

In that cause you should have named the testmethodes the same way. Dont test both methods in one testcase.

You also need a seconds testcase for each method too verify that the Exception is thrown by your service when the Stelle with the id doenst exists